packages/mds-transactions-service/@types/index.ts

```diff
@@ -1,27 +1,134 @@
 import { DomainModelCreate } from '@mds-core/mds-repository'
 import { RpcServiceDefinition, RpcRoute } from '@mds-core/mds-rpc-common'
+import { Nullable, Timestamp, UUID, VEHICLE_TYPE, VehicleEvent } from '@mds-core/mds-types'
 
-export interface TransactionDomainModel {
-  name: string
-  text: string
+// one example -- many others are possible
+export interface TripReceiptDetailsDomainModel {
+  trip_id: UUID
+  start_timestamp: Timestamp
+  end_timestamp: Timestamp
+  vehicle_type: VEHICLE_TYPE
+  start_geography_id: Nullable<UUID>
+  end_geography_id: Nullable<UUID>
+  duration: Number // seconds
+  distance: Number // meters
+  trip_events: VehicleEvent[]
+}
+
+export interface CurbUseDetailsDomainModel {
+  trip_id: UUID
+  start_timestamp: Timestamp
+  end_timestamp: Timestamp
+  vehicle_type: VEHICLE_TYPE
+  geography_id: Nullable<UUID>
+  duration: Number // seconds
+  trip_events: VehicleEvent[]
+}
+
+export const FEE_TYPES = [
+  'base_fee',
+  'upgrade_fee',
+  'congestion_fee',
+  'trip_fee',
+  'parking_fee',
+  'reservation_fee',
+  'distance_fee',
+  'tolls_fee'
+] as const
+
+export type FEE_TYPE = typeof FEE_TYPES[number]
+
+export interface ReceiptDomainModel {
+  receipt_id: UUID
+  timestamp: Timestamp // could be any time before the Transaction was created
+  origin_url: URL // where can I go to dig into the details of the receipt, given this receipt_id?
+  receipt_details: Object // JSON blob with free-form supporting evidence, DO NOT INCLUDE PII
```

Comment:
Assuming things like `CurbUseDetailsDomainModel` and `TripReceiptDetailsDomainModel` will be options here? Could just throw them into a union as a placeholder for now, e.g. `receipt_details: Object | CurbUseDetailsDomainModel | TripReceiptDetailsDomainModel`